index.js

```diff
@@ -192,7 +88,73 @@ module.exports = function (config) {
         callback(null, json)
       })
     }
+  }
+
+  /**
+   * express middleware setup
+   */
+
+  // save the koop log onto the app
+  app.log = koop.log
```

Comment:
I don't think `app.log` is actually getting used anywhere. @dmfenton can I just take this out?